app/views/admin/loans/index.html.slim

```diff
@@ -2,6 +2,8 @@
 
 - content_for :with_title
   = link_to t("loan.shared.new"), new_admin_loan_path, class: 'btn btn-default new-button'
+  - if policy(::Accounting::Transaction).update?
```

Comment:
Since the updater is currently only updating active loans, should we also only show this link if the loan is active?

Reply:
This is on the loan index, so it isn't per-loan.

Reply:
ah ok, thanks.